testutil/dbustest.go

```diff
@@ -54,6 +54,7 @@ func (s *DBusTest) SetUpSuite(c *C) {
 	err := s.dbusDaemon.Start()
 	c.Assert(err, IsNil)
 	s.oldSessionBusEnv = os.Getenv("DBUS_SESSION_BUS_ADDRESS")
+	os.Unsetenv("DBUS_SESSION_BUS_ADDRESS")
```

Comment:
Shouldn't we do:
```
os.Setenv("DBUS_SESSION_BUS_ADDRESS", fmt.Sprintf("unix:%s/user_bus_socket", s.tmpdir))
```
now?

Reply:
It takes a while for the socket to appear,  http://paste.ubuntu.com/p/mwvRRjjqzf/